### PR TITLE
Improving OAuth one step at a time.

### PR DESCRIPTION
**Pull request by jermenkoo**
## What does this PR do?
For example: Changes the way we retrieve bookings.

## ✅ Pull Request checklist

- [x] Is this code complete?
- [x] Are tests done/modified?

## 🚨 Is this a breaking change for API clients?
No

## :squirrel: Deploy notes
For example: Need to run migrations as part of deployment

### CODE REVIEW

**Comment by ChristopherHammond13** [comment]
I evened this up with master. I'm going to try and build statics myself but if that doesn't work I might need someone else here to do that prior to merging.

**Comment by ChristopherHammond13** [comment]
Can someone build statics on this? If so I'll merge.

**Comment by wilhelmklopp** [comment]
@ChristopherHammond13 done

**Comment by ChristopherHammond13** [comment]
Merging now

File: frontend/src/react/pages/authorise.jsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ class AuthoriseApp extends React.Component {
               </ul>
             </div>
             <form method="post" action="/oauth/user/allow" className={formButton}>
-              <button type="submit" className="pure-button pure-button-primary padded">Authorise hackathon</button>
+              <button type="submit" className="pure-button pure-button-primary padded">Authorise {appName}</button>
```
Comment (jermenkoo) [inline frontend/src/react/pages/authorise.jsx]: `{this.state.data.app_name}`?

Comment (ChristopherHammond13) [reply frontend/src/react/pages/authorise.jsx]: ah, that looks like it could solve it

**Pull request merged**